Code/Tools/FBuild/FBuildCore/Cache/CachePlugin.cpp

```diff
@@ -64,20 +74,18 @@
 
 // GetFunction
 //------------------------------------------------------------------------------
-void * CachePlugin::GetFunction( const char * friendlyName, const char * mangledName ) const
+void * CachePlugin::GetFunction( const char * name) const
 {
     #if defined( __WINDOWS__ )
         ASSERT( m_DLL );
-        void * func = ::GetProcAddress( (HMODULE)m_DLL, mangledName );
+        void * func = ::GetProcAddress( (HMODULE)m_DLL, name );
         if ( !func )
         {
             FLOG_WARN( "Missing CachePluginDLL function '%s' (Mangled: %s)", friendlyName, mangledName );
```

Comment:
This won't compile anymore on Windows - you deleted friendlyName and mangledName